tools/osbuilder/node-builder/azure-linux/README.md

```diff
@@ -174,6 +174,9 @@ This section describes how to build and deploy in debug mode.
    release mode, or `debug` to build it in debug mode.
  * `AGENT_POLICY_FILE`: Specify `allow-set-policy.rego` (default) to use
    a restrictive policy, or `allow-all.rego` to use a permissive policy.
+ * `UVM_BUILD_TYPE`: Specify `release` (default) to build the UVM with
```

Comment:
We should talk here. Let's potentially avoid intermingling this variable with the meaning of debug builds in this section. Maybe we have to rework the section a little bit.